minkyu/app.js

```diff
@@ -31,24 +30,57 @@ app.use(cors());
 app.use(morgan('tiny'));
 
 app.get("/ping", (req,res) =>{
-    res.status(200).json({"message" : "pong"});
+    res.json({message : "pong"});
```

Comment:
```suggestion
    res.status(200).json({message : "pong"});
```
백엔드에서 status 코드 또한 handling이 가능합니다. 적절한 status code를 추가해주세요 !